src/azure-cli/azure/cli/command_modules/appconfig/__init__.py

```diff
@@ -0,0 +1,30 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.core import AzCommandsLoader
+
+import azure.cli.command_modules.appconfig._help  # pylint: disable=unused-import
+
+
+class AppconfigCommandsLoader(AzCommandsLoader):
+
+    def __init__(self, cli_ctx=None):
+        from azure.cli.core.commands import CliCommandType
+        configstore_custom = CliCommandType(
+            operations_tmpl='azure.cli.command_modules.appconfig.custom#{}')
+        super(AppconfigCommandsLoader, self).__init__(
+            cli_ctx=cli_ctx, custom_command_type=configstore_custom)
```

Comment:
Please update this to suppress your extension:
https://github.com/Azure/azure-cli/blob/dev/doc/authoring_command_modules/authoring_commands.md#extension-suppression

Reply:
Added